Makefile

```diff
@@ -59,6 +59,26 @@ test-go: ## Runs all golang tests
 test-sh: ## Runs all shellscript tests
 	./hack/test-sh.sh
 
+##@ Tools
+
+RELEASE_TOOLS ?=
+
+.PHONY: release-tools
+
+release-tools: ## Compiles a set of release tools, specified by $RELEASE_TOOLS
+	./compile-release-tools $(RELEASE_TOOLS)
+
+##@ GCB Jobs
+
+.PHONY: stage-ci
+
+stage-ci: ## Compiles/installs krel and submits a streaming stage build to GCB (used for Prow)
```

Comment:
can we mention this is a mock one? or everybody knows that is always a mock when don't have the `--nomock` flag

Reply:
@cpanato -- Good point! I just updated it to point out this is a mock build. :)

Reply:
cool! thanks!